cmd/commandfuncs.go

```diff
@@ -208,6 +208,10 @@ func cmdRun(fl Flags) (int, error) {
 		}
 	}
 
+	if err := sdNotify("READY=1"); err != nil {
+		caddy.Log().Error("unable to notify systemctl", err)
+	}
```

Comment:
This is before the config is loaded.

Reply:
Apologies? I didn't understand the comment here. Are you saying we should `sdNotify` before the config is ready? If yes, https://github.com/caddyserver/caddy/issues/3876#issue-747061574 says "when initial config is properly loaded". So I kept it after the config was loaded.

Reply:
`loadConfig` just parses the config, it doesn't actually "run" it. That happens next, with `caddy.Load(config, true)`. So it should be after that.

Reply:
Yep, exactly. Thanks!